src/Soil-Core-Tests/SoilTest.class.st

```diff
@@ -97,7 +97,7 @@ SoilTest >> testSerializingRootTwoCluster [
 	tx checkpoint.
 	tx2 := soil newTransaction.
 	materializedRoot := tx2 root.
-	self assert: materializedRoot nested reference class equals: SOObjectProxy.
+	self assert: materializedRoot nested reference class equals: SOTestClusterRoot.
```

Comment:
This is wrong. The nested reference returns a proxy and even ProtoObject can answer class